commons/src/main/java/com/powsybl/commons/report/ReportNodeBuilder.java

```diff
@@ -7,12 +7,31 @@
  */
 package com.powsybl.commons.report;
 
+import java.time.format.DateTimeFormatter;
+
 /**
  * A builder to create a {@link ReportNode} object.
  *
  * @author Florian Dupuy {@literal <florian.dupuy at rte-france.com>}
  */
 public interface ReportNodeBuilder extends ReportNodeAdderOrBuilder<ReportNodeBuilder> {
+
+    /**
+     * Enable/disable timestamps on build ReportNode and all descendants
+     * @param timestampPattern: the pattern to use for the timestamp (see {@link DateTimeFormatter#ofPattern}), or null to disable the timestamp
+     * @return a reference to this object
+     */
+    ReportNodeBuilder withTimestamps(String timestampPattern);
```

Comment:
I would have add 2 methods, one for activation or not the timestamped logging and another to change the default date pattern.